rio_tiler/io/cogeo.py

```diff
@@ -554,11 +566,18 @@ def feature(
         if not dst_crs:
             dst_crs = shape_crs
 
+        vrt_options = kwargs.pop("vrt_options", {})
+
         # Get BBOX of the polygon
         bbox = featureBounds(shape)
 
+        # If Dataset Origin is top Left (non-geo) we need to invert the bbox
+        # And set src_method="NO_GEOTRANSFORM" to use WarpedVRT
+        if self.bounds[1] > self.bounds[3] and not self.crs:
```

Comment:
I think we could reuse `_normalize_bounds` here.

Reply:
No! because we want the opposite. 

the issue here is that bbox will be in form of `[minx, miny, maxx, maxy]` (output of `featureBounds(shape)`) but we want to bbox to be in form of `[minx, maxy, maxx, miny]` because the origin of non-geo data is the upper left